### ISSUE TITLE
[TF2] Incorrect update_url in mainmenuoverride.res

### ISSUE THREAD

**Issue by Tiagoquix** [ValveSoftware/Source-1-Games#3217]
In the `mainmenuoverride.res` file, the `update_url` parameter points the user to 
```
http://store.steampowered.com/news/?filter=updates&appids=440
```
but this is incorrect since if you follow that link you will see all news for Team Fortress 2, and not only the updates.
The link should be
```
http://store.steampowered.com/news/?appids=440&appgroupname=Team+Fortress+2&feed=steam_updates
```
and then the link will be the correct link.

**Comment by Yetoo1** [comment]
Why did you close this?

**Comment by Tiagoquix** [comment]
> @Tiagoquix Why did you close this?

Just test the links. Apparently, over time, Steam did some changes to the News page and now the link redirects to the correct page.

If you use the current link present in mainmenuoverride.res, it will redirect to the new News page.

The new link I suggested redirects to the old News page.

Since the purpose was to make the link redirect to the new News page, this is not a problem anymore.

**Comment by Yetoo1** [comment]
@Tiagoquix Yes, while the link you suggested redirects to old news, the original link still shows all news instead of just updates as stated in the bug report. The original link doesn't show all smaller updates before November or whenever the new news page was introduced and still shows news posts that are more than updates (albeit any content requires a user to use a filter to see, but there is no filter for just content updates). Yes, I understand that due to the changes in steam there is no way the developers of TF2 are going to change the update string to an older version of steam news and instead try to keep up with the changes of steam, but if one were going by the direct words of the bug report, it's still technically an issue since a more direct page exists. A developer would need to mark as won't fix or delineate that the new news link is supposed to do that functionality but it is not implemented yet if the bug were to be closed.

**Comment by Tiagoquix** [comment]
Retested today. Link shows only updates, so it works as intended. Closing.
